utils/xlsx-utility.js

```diff
@@ -1,20 +1,22 @@
 'use strict';
-var XLSX = require('xlsx');
+var xlsx = require('xlsx');
 var bluebird = require('bluebird');
-var s3 = require('../utils/s3-utility.js');
+var s3 = require('./s3-utility.js');
+var mongoUtil = require('./mongo-utility.js');
 
 function parseFile(fileName) {
-	var workbook = XLSX.readFile('./uploads/' + fileName);
-	var worksheets = [];
-	var sheetJSON = [];
-	var sheetNameList = workbook.SheetNames;
-	var i = 0;
-	sheetNameList.forEach(function(y){
-		worksheets[i] = workbook.Sheets[y];
-		sheetJSON[i] = XLSX.utils.sheet_to_json(worksheets[i]);
+  var workbook = xlsx.readFile('./uploads/' + fileName);
+  var worksheets = [];
+  var sheetJSON = [];
+  var sheetNameList = workbook.SheetNames;
+  var i = 0;
+  sheetNameList.forEach(function(y){
+    worksheets[i] = workbook.Sheets[y];
+    sheetJSON[i] = xlsx.utils.sheet_to_json(worksheets[i]);
     bluebird.each(sheetJSON[i], function(jsonObject) {
       if (jsonObject['Category ID'] && jsonObject.Category) {
         console.log(jsonObject);
```

Comment:
Do not push any console.log / debug codes to git. Keep it in you local for the development purpose only.